### PR TITLE
Alpha/fonts-update

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Added documentation page for Headings
* Updated Headings font styles

captura de tela 2017-04-19 as 10 47 02 am

### CODE REVIEW

File: docs/demo/gaiden-css/generic/headings/demo.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+  </style>
+  <title>Headings</title>
+</head>
+
+<body>
+    <h1 class="" arial-level="1" role="heading">
+      H1 Title Gaiden
+      <small>Sub Title Gaiden</small>
```
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/generic/headings/demo.html]: Não é um pouco ruim esse tipo de marcação. Não seria interesante nós utilizarmos um tag de sub title dado que é um sub title?

File: src/scss/generic/_heading.scss [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 .page-title,
 .section-title,
 .container-title,
 .card-title,
 .info-title {
-  color: get-color(blackNeutral);
+  color: get-color(rastaGray);
   font: {
-    weight: 700;
+    weight: 900;
```
Comment (MarcoBrunoBR) [inline src/scss/generic/_heading.scss]: Nesse ponto não vale criar um variável como bold, ligth e ....?
Já que estamos utilizando sass podemos aproveitar e deixar o código mais expressivo. :-)

File: src/scss/generic/_heading.scss [hunk 3]
```diff
@@ -33,59 +38,64 @@ h5,
...
 h1,
 .page-title {
-  @include media(small) {
-    font-size: 36px;
-  }
-
   margin-bottom: 4px;
   font-size: 24px;
 
   small {
```
Comment (MarcoBrunoBR) [inline src/scss/generic/_heading.scss]: Temos alguma regra para usar ou não usar seletores de tag?

File: docs/demo/gaiden-css/generic/headings/demo.html (hunk 1 above)
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/generic/headings/demo.html]: Não é um pouco ruim esse tipo de marcação. Não seria interesante nós utilizarmos um tag de sub title dado que é um sub title?

File: src/scss/generic/_heading.scss (hunk 2 above)
Comment (MarcoBrunoBR) [inline src/scss/generic/_heading.scss]: Nesse ponto não vale criar um variável como bold, ligth e ....?
Já que estamos utilizando sass podemos aproveitar e deixar o código mais expressivo. :-)

File: src/scss/generic/_heading.scss [hunk 4]
```diff
@@ -33,59 +38,64 @@ h5,
...
     margin-top: 4px;
-    font-size: 14px;
+    font-size: 16px;
+  }
+
+  @include media(normal) {
+    font-size: 24px;
+
+    small {
```
Comment (MarcoBrunoBR) [inline src/scss/generic/_heading.scss]: Temos alguma regra para usar ou não usar seletores de tag?

Comment (iondrimba) [inline src/scss/generic/_heading.scss]: @MarcoBrunoBR No rules were defined for sass linting yet.

File: docs/demo/gaiden-css/generic/headings/demo.html (hunk 1 above)
Comment (iondrimba) [inline docs/demo/gaiden-css/generic/headings/demo.html]: @MarcoBrunoBR I don't think this approach is a bad practice. What tag do you have in mind?

Comment (iondrimba) [inline docs/demo/gaiden-css/generic/headings/demo.html]: @MarcoBrunoBR also changing a tag is a big refactor. You would have to change every occurency of all titles inside every project that uses Gaiden.

File: src/scss/generic/_heading.scss (hunk 2 above)
Comment (iondrimba) [inline src/scss/generic/_heading.scss]: @MarcoBrunoBR good suggestion 👍

File: docs/demo/gaiden-css/generic/headings/demo.html (hunk 1 above)
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/generic/headings/demo.html]: @iondrimba 
h1 => title
h2 => subtitle

Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/generic/headings/demo.html]: @iondrimba, that's why it's not cool to use tag selectors.

Comment (iondrimba) [inline docs/demo/gaiden-css/generic/headings/demo.html]: @MarcoBrunoBR I agree with you on that point of tag selectors 👍

Comment (iondrimba) [inline docs/demo/gaiden-css/generic/headings/demo.html]: h1–h6 elements must not be used to markup subheadings, subtitles, alternative titles and taglines unless intended to be the heading for a new section or subsection. https://www.w3.org/TR/html5/common-idioms.html

Comment (eduardojmatos) [inline docs/demo/gaiden-css/generic/headings/demo.html]: `H(n)` are not about titles and subtitles. They are relative to the levels of headings on a document or section.

About tags and styles, in this case, `small` is a semantic way to declare that a piece of content on heading is not so important like the principal content. So, in my opinion, we shouldn't use another tag inside a heading to declare this kind of division of a content than `small`.

File: src/scss/generic/_heading.scss (hunk 2 above)
Comment (eduardojmatos) [inline src/scss/generic/_heading.scss]: I think we need to have more than one case to export values to variables or maps. If there is more than one case, let's move to a map and use some function like `get-weight(bold)`.

File: docs/demo/gaiden-css/generic/headings/demo.html (hunk 1 above)
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/generic/headings/demo.html]: If we have:
<Section>
<H1> Tittle </ h1>
<H2> Subtitle </ h2>
</ Section>

For h2 without less relevant than h1 this no longer defines as subtitle?
I think it's worth seeing this with the SEO guys.

File: src/scss/generic/_heading.scss (hunk 2 above)
Comment (MarcoBrunoBR) [inline src/scss/generic/_heading.scss]: Nice @eduardojmatos :-)

File: src/scss/tools/mixins/_get-weight.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+@function get-weight($kind) {
```
Comment (iondrimba) [inline src/scss/tools/mixins/_get-weight.scss]: @MarcoBrunoBR @eduardojmatos added font-weight helper

File: src/scss/settings/_base_font.scss [hunk 6]
```diff
@@ -2,3 +2,9 @@
 @import url('//fonts.googleapis.com/css?family=Lato:400,400i,700,700i,900,900i');
 
 $baseFont: 'Lato';
+
+$fontWeight: {
```
Comment (iondrimba) [inline src/scss/settings/_base_font.scss]: @MarcoBrunoBR @eduardojmatos are you ok with these key names?

File: docs/demo/gaiden-css/generic/headings/demo.html (hunk 1 above)
Comment (eduardojmatos) [inline docs/demo/gaiden-css/generic/headings/demo.html]: Looking your example I don't see an association with implementation of classes and small.

To clarify, h2-6 exists to be a sublevel of heading, not to define visually a subtitle. Our category page have a little "caption" on H1, but this information is not a subtitle of section.

File: src/scss/settings/_base_font.scss (hunk 6 above)
Comment (eduardojmatos) [inline src/scss/settings/_base_font.scss]: :shipit:

**Pull request merged**